docs/index.rst

```diff
@@ -1,17 +1,67 @@
 .. include:: ../README.rst
    :end-before: About
 
+
 .. include:: ../README.rst
-   :start-after: -----
+   :start-line: 25
    :end-before: Features
 
-.. toctree::
-   :numbered:
 
-   introduction
-   cli
-   restapi
-   contributing
-   changes
-   license
-   authors
+.. include:: ../README.rst
+   :start-line: 32
+   :end-before: Useful links
+
+**Contents**
+
+.. contents::
+   :local:
+   :depth: 1
+   :backlinks: none
+
+
+CLI API
+=======
+
+.. click:: reana_server.reana_admin:reana_admin
+   :prog: flask reana-admin
+   :show-nested:
+
+
+REST API
+========
+
+The REANA Server offers a REST API for management workloads
+(workflows, jobs, tasks, etc.) running on REANA Cloud.
+Detailed REST API documentation can be found `here <_static/api.html>`_.
+
+.. automodule:: reana_server.rest.ping
+   :members:
+
+.. automodule:: reana_server.rest.users
+   :members:
+
+.. automodule:: reana_server.rest.workflows
+   :members:
+
+
+.. include:: ../CONTRIBUTING.rst
+
+.. include:: ../CHANGES.rst
+
+License
+=======
+
+.. include:: ../LICENSE
+
+In applying this license, CERN does not waive the privileges and immunities
+granted to it by virtue of its status as an Intergovernmental Organization or
+submit itself to any jurisdiction.
+
+.. include:: ../AUTHORS.rst
+
+
+Useful links
+============
+
+.. include:: ../README.rst
```

Comment:
Hmm, the `documentation` link from README.rst would be self-referencing.  We should probably amend the link to point to docs.reana.io... Or simply kill the thing here in rtfd and end with author list. That's probably better. This is because people who are here already know where the source code is, and we link to issues above, and list of releases is not terribly interesting since end users would use Helm releases anyway... So let's end with author list I guess.

Reply:
Do you mean removing the whole "Useful links" section or only the bullet-point links?